apps/synapse-oauth-signin/package.json

```diff
@@ -66,7 +66,7 @@
     "markdown-it-synapse-math": "^3.0.4",
     "markdown-it-synapse-table": "^1.0.6",
     "memfs": "^3.4.7",
-    "msw": "1.0.1",
```

Comment:
Upgrading MSW past v1 in SRC seems to break a few things. Let's use the same version (0.49.3) across the monorepo